Library/Homebrew/os/mac.rb

```diff
@@ -47,19 +47,15 @@ def cat
     end
 
     def languages
-      return @languages unless @languages.nil?
+      return @languages if defined?(@languages)
 
       @languages = Utils.popen_read("defaults", "read", ".GlobalPreferences", "AppleLanguages").scan(/[^ \n"(),]+/)
 
-      if ENV["HOMEBREW_LANGUAGES"]
-        @languages = ENV["HOMEBREW_LANGUAGES"].split(",") + @languages
-      end
-
-      if ARGV.value("language")
-        @languages = ARGV.value("language").split(",") + @languages
-      end
-
-      @languages = @languages.uniq
+      @languages = [
```

Comment:
`@languages.prepend` or something feels nicer?

Reply:
Doesn't really feel nicer to me.

Reply:
It feels a little weird to be doing a splat on `@languages` inside it's own assignment.

Reply:
It's now all in one assignment.